src/server/db/booking-schema.ts

```diff
@@ -0,0 +1,30 @@
+import { relations } from "drizzle-orm";
+import { pgTable, serial, text, timestamp } from "drizzle-orm/pg-core";
+import { user } from "./auth-schema";
+
+export const bookings = pgTable("bookings", {
+  id: serial("id").primaryKey(),
+  title: text("title").notNull(),
+  pickupLocation: text("pickup_location").notNull(),
+  dropoffLocation: text("dropoff_location").notNull(),
+  purpose: text("purpose"),
+  passengerInfo: text("passengerInfo").notNull(),
+  status: text("status", { enum: ["incomplete", "completed", "in-progress"] })
+    .notNull()
+    .default("incomplete"),
+  driverId: text("driver_id")
+    .notNull()
+    .references(() => user.id),
```

Comment:
setting the driver to null if the driver object is deleted
if the agency is deleted (the one who made the booking) the form should also be deleted in that case

```suggestion
  // the agency that created the booking
  agencyId: text("agency_id")
    .notNull()
    .references(() => user.id, { onDelete: "cascade" }),

  driverId: text("driver_id")
    .references(() => user.id, { onDelete: "set null" }), 
```